dxr/query.py

```diff
@@ -189,6 +197,39 @@ def direct_result(self):
                 elif len(results) > 1:
                     return None
 
+    def _perform_search(self, filters, is_line_query, offset, limit, query=None, sort=True):
+        """Perform a query using self.es_search, returning a tuple (total count of results,
+        list of results)
+
+        :arg filters: nested list of filters
+        :arg is_line_query: whether to query lines (otherwise query files)
+        :arg offset: return results starting from this number
+        :arg limit: maximum number of results to return
+        :arg query: valid ES query; if omitted then construct one from filters
+        :arg sort: whether to sort the results alphabetically by path and numerically by line
```

Comment:
It always sorts by something, so maybe something like `sort_by_path` or `score` or `alphabetize` would be better?